.local.env

```diff
@@ -1,14 +1,18 @@
 NODE_ENV=development
 
+# Core cube API
 API_CORE_BASE=https://cube-creator.lndo.site/
-VUE_APP_API_URL=https://cube-creator.lndo.site/
 
+# Vue UI app
+VUE_APP_API_URL=https://cube-creator.lndo.site/
 UI_BASE=https://app.cube-creator.lndo.site/
 
+# SPARQL endpoints
 STORE_QUERY_ENDPOINT=http://store:3030/cube-creator/query
 STORE_UPDATE_ENDPOINT=http://store:3030/cube-creator/update
 STORE_GRAPH_ENDPOINT=http://store:3030/cube-creator/data
 
-AUTH_AUDIENCE=https://datacube.zazukoians.org/
+# OIDC config
+AUTH_AUDIENCE=cube-creator
```

Comment:
Do we need a new client in Keycloak to have a proper audience here? @sandhose

Reply:
Yup; I think the instructions are up to date here: https://hackmd.io/rW_4v3BsRQS04DKDLKYpDQ#Full-stack-app-check-list